packages/blocks/table/src/table.tsx

```diff
@@ -124,9 +124,9 @@ export const Table: BlockComponent<BlockEntityProperties> = ({
 
     return [
       {
-        pageCount: aggregate?.pageCount || 1,
-        pageNumber: aggregate?.pageNumber || 1,
-        pageSize: aggregate?.itemsPerPage || 1,
+        pageCount: aggregate?.pageCount ?? 1,
+        pageNumber: aggregate?.pageNumber ?? 1,
+        pageSize: aggregate?.itemsPerPage ?? 1,
```

Comment:
I'm not sure this is the intended behavior – we don't want to set `pageNumber` or `itemsPerPage` as `0`, as this will result in no results and seems pointless. I don't think it really matters as you can't set either to `0` in the UI.

Reply:
I'll revert this back and add disable comments